src/main/java/com/algorand/algosdk/v2/client/model/Account.java

```diff
@@ -0,0 +1,311 @@
+	public Long getPendingRewards(){
+		return pendingRewardsIsSet ? pendingRewards : null;
+	}
+	/**
+	 * Check if has a value for pendingRewards 
+	 */	@JsonIgnore
+	public boolean hasPendingRewards(){
+		return pendingRewardsIsSet;
+	}
+
+	/**
+	 * (ebase) used as part of the rewards computation. Only applicable to accounts 
+	 * which are participating. 
+	 */
+	private long rewardBase;
+	private boolean rewardBaseIsSet;
+	@JsonProperty("reward-base")
+	public void setRewardBase(long rewardBase){
+		this.rewardBase = rewardBase;
+		rewardBaseIsSet = true;
+	}
+	@JsonProperty("reward-base")
+	public Long getRewardBase(){
+		return rewardBaseIsSet ? rewardBase : null;
+	}
+	/**
+	 * Check if has a value for rewardBase 
+	 */	@JsonIgnore
+	public boolean hasRewardBase(){
+		return rewardBaseIsSet;
+	}
+
+	/**
+	 * (ern) total rewards of MicroAlgos the account has received, including pending 
+	 * rewards. 
+	 */
+	private long rewards;
+	private boolean rewardsIsSet;
+	@JsonProperty("rewards")
+	public void setRewards(long rewards){
+		this.rewards = rewards;
+		rewardsIsSet = true;
+	}
+	@JsonProperty("rewards")
+	public Long getRewards(){
+		return rewardsIsSet ? rewards : null;
+	}
+	/**
+	 * Check if has a value for rewards 
+	 */	@JsonIgnore
+	public boolean hasRewards(){
+		return rewardsIsSet;
+	}
+
+	/**
+	 * The round for which this information is relevant. 
+	 */
+	private long round;
+	private boolean roundIsSet;
+	@JsonProperty("round")
+	public void setRound(long round){
+		this.round = round;
+		roundIsSet = true;
+	}
+	@JsonProperty("round")
+	public Long getRound(){
+		return roundIsSet ? round : null;
+	}
+	/**
+	 * Check if has a value for round 
+	 */	@JsonIgnore
+	public boolean hasRound(){
+		return roundIsSet;
+	}
+
+	/**
+	 * (onl) delegation status of the account's MicroAlgos * Offline - indicates that 
+	 * the associated account is delegated. * Online - indicates that the associated 
+	 * account used as part of the delegation pool. * NotParticipating - indicates that 
+	 * the associated account is neither a delegator nor a delegate. 
+	 */
+	private String status;
+	private boolean statusIsSet;
+	@JsonProperty("status")
+	public void setStatus(String status){
+		this.status = status;
+		statusIsSet = true;
+	}
+	@JsonProperty("status")
+	public String getStatus(){
+		return statusIsSet ? status : null;
+	}
+	/**
+	 * Check if has a value for status 
+	 */	@JsonIgnore
+	public boolean hasStatus(){
+		return statusIsSet;
+	}
+
+	/**
+	 * Indicates what type of signature is used by this account, must be one of: * sig 
+	 * * msig * lsig 
+	 */
+	private String type;
+	private boolean typeIsSet;
+	@JsonProperty("type")
+	public void setType(String type){
+		this.type = type;
+		typeIsSet = true;
+	}
+	@JsonProperty("type")
+	public String getType(){
+		return typeIsSet ? type : null;
+	}
+	/**
+	 * Check if has a value for type 
+	 */	@JsonIgnore
+	public boolean hasType(){
+		return typeIsSet;
```

Comment:
You can return `type != null` here instead of having a flag for each type. Also it looks like there is a missing newline before `@JsonIgnore`.

Reply:
fixed.